examples/simple.js

```diff
@@ -132,31 +140,41 @@ d3.tsv("../data/PCA_transcript_expression_TMM_RPKM_log2_sample_data.6932.tsv", f
       data.forEach(function(d) {
         d.PC1 = +d.PC1;
         d.PC2 = +d.PC2;
+        d.PC3 = +d.PC3;
+        d.PC4 = +d.PC4;
+        d.PC5 = +d.PC5;
       });
 
       target = rootDiv;
       var options = {
         metadata: metadata,
         clickedBars: clickedBarId,
+        numOfClickedBars: numOfClickedBars,
         numberOfComponents: numberOfComponents,  //used to determine how many components will be showed in the bar chart
         barChartData: barChartData,
-        barChartHeight:480,
-        barChartWidth: numberOfComponents * 30,
+        barChartHeight:500,
+        barChartWidth: numberOfComponents * 40,
         groupByoption: groupByoption,
         xDomain: "PC1",
```

Comment:
The default graph uses PC1 and PC2, so I just hardcode these values for convenience. I will modify it later to use variables.